index.js

```diff
@@ -0,0 +1,33 @@
+/* global hexo */
+
+'use strict';
+
+hexo.config.math = Object.assign({
+  katex: {
+    css: 'https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css',
+    options: {
+      throwOnError: false
+    }
+  },
+  mathjax: {
+    css: 'https://cdn.jsdelivr.net/gh/hexojs/hexo-math/dist/style.css',
```

Comment:
Before we releasing a newer version we should change it to

```
https://cdn.jsdelivr.net/npm/hexo-math@x.y.z/dist/style.css
```

With a specific version jsDelivr will send `cache-control: public, max-age=35136000, immutable`.